src/libraries/sendtohelix.proj

```diff
@@ -97,7 +102,20 @@
   <Target Name="BuildHelixWorkItems"
           DependsOnTargets="CompressRuntimeDirectory">
 
-    <ItemGroup>
+    <ItemGroup Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS'">
+      <!-- Find all directories named *.app and strip the trailing slash -->
+      <_appBundleFiles Include="$(TestArchiveTestsRoot)**/*"/>
+      <_appBundleDirs Include="@(_appBundleFiles->'%(RootDir)%(Directory)'->Distinct())"/>
+      <XHarnessAppFolderToTest Include="@(_appBundleDirs->Replace(&quot;.app/&quot;, &quot;.app&quot;))"
+                               Condition="$([System.Text.RegularExpressions.Regex]::IsMatch(%(Identity), '\.app/?$'))">
```

Comment:
**Help wanted:**
Can someone please let me know if there's a better way to find all directories ending with `.app`?
Or should I just hardcode the path inside the test assets artifact that contains the folders?

The trailing `/` will be fixed in Helix SDK later

Reply:
Suggestions from @akoeplinger on internal chat:

should be clearer with `<XHarnessAppFolderToTest Include="@(_appBundleDirs)" Condition="%(Identity.EndsWith('.app')) or %(Identity.EndsWith('.app/'))` I think
assuming these directories are at the edge of the tree, rather than somewhere in the middle
or even easier: `<XHarnessAppFolderToTest Include="$([System.IO.Directory]::GetDirectories('$(TestArchiveTestsRoot)', '*.app', System.IO.SearchOption.AllDirectories))" />`